tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/Operate.kt

```diff
@@ -0,0 +1,20 @@
+package net.corda.cli.plugins.network
+
+import picocli.CommandLine.Command
+
+@Command(
+    name = "operate",
+    description = [
+        "Operate on the network",
+        "This sub command should only be used for internal development"
+    ],
+    subcommands = [
+        AllowClientCertificate::class,
+        ExportGroupPolicy::class
+    ]
+)
+class Operate : Runnable {
+    override fun run() {
+        // Implementation for the operate command
+    }
+}
```

Comment:
This can be as below since `operate` won't be a standalone command:
```suggestion
class Operate
```